html/changelog.html

```diff
@@ -123,6 +127,27 @@ <h3 class="author">itseasytosee updated:</h3>
 			<h3 class="author">necromanceranne updated:</h3>
 			<ul class="changes bgimages16">
 				<li class="bugfix">Fixes regal pipeguns inheriting jamming from pipeguns.</li>
+			</ul>
+
+			<h2 class="date">20 February 2021</h2>
```

Comment:
Remove your changelog changes.

Reply:
Fixed - this hopefully will be the last PR with my messed up master branch.